walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,50 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Comparator;
+import java.util.Arrays;
+import java.util.Iterator;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+  public Collection<TimeRange> query (Collection<Event> events, MeetingRequest request) {
+      Collection<String> people = request.getAttendees();
+      Collection<TimeRange> availableTimes = new ArrayList<TimeRange>();
```

Comment:
we can omit the type parameter

Collection<TimeRange> availableTimes = new ArrayList<>();

Reply:
done I think